.github/labels.md

```diff
@@ -0,0 +1,234 @@
+create the installer. ??? I see an overlap with `component:build system` here.
+
+#### Label `component:documentation`
+
+The Issue requires knowledge about Doxygen or primarly contains documentation.
+
+#### Label `component:game mechanic`
+
+The Pull Request requires knowledge about the currently used or planned game
+mechanics.  The content of the Pull Request may require extensive gameplay
+balance testing.
+
+#### Label `component:infrastructure`
+
+The Pull Request requires knowledge about non-game related infrastructure that
+aids development.  For example this includes issues with the project homepage,
+used external services like continous integration or hosting of release
+binaries.
+
+#### Label `component:internal`
+
+??? No idea.  Thread suggest it's a non-user visible feature, but then it's the
+wrong category.
+
+#### Label `component:network`
+
+The Issue requires knowledge about TCP/IP networking and establishing a test
+network infrastructure.
+
+#### Label `component:scripting`
+
+The Pull Request requires knowledge about the FOCS language and probably game
+mechanics.  Probably most used in conjunction with `component:game mechanic` and
+`component:art music text assets`.
+
+#### Label `component:translation`
+
+The Pull Request requires knowledge about the game style, game terminology,
+stringtable syntax and the translation process in general. It maybe helpful to
+know the language of the translation to proofread.
+
+#### Label `component:UI`
+
+The Pull request requires knowledge about the game style, the expected user
+experience and graphical APIs used (GiGi and OpenGL).
+
+#### Label `status:cherry-pick for release`
+
+The Pull Request should be applied to the currently open release branch.  Use
+this label sparsely, only for bug fixes, and document a minimal set of commits
+that should be considered by the release manager for cherry-picking.
+Cherry-picking Pull Requests onto release branches should fix bugs, not
+introduce new ones.
+
+#### Label `status:duplicate`
+
+The topic of the Issue is or was already adressed by another Issue.  Add the
+number of the adressing Issue in a comment and close this Issue.  If the
+closed Issue contains new knowledge that is valuable to solve the adressing
+Issue copy it over to the adressing Issue.
+
+#### Label `status:follow up`
+
+The Pull Request discussion showed that the proposed implementation is
+insufficient in some way or the Pull Request in itself is malformed.  Usually
+not required as we prefer rebasing within the same Pull request to update
+the implementation.  When applying this Label reference the new Pull Request
+number in the old one.
+
+#### Label `status:help wanted`
+
+??? I don't think we really need that.  At least I didn't see any Issue were it
+caused someone required to help to magically appear.  This can be handled better
+with `@`-mentions if someones or some groups special knowledge is required.
+
```

Comment:
I took the general description by @LGM-Doyle about `help wanted`.

And yes, @Vezzra has a point here.  If someone is opening a PR he wants a review or a review is required because it's an external contributor.